python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -444,32 +494,46 @@ async def call_pipetask(self, image_type, exposure_ids_dict):
             pipe_yaml = "findDefects.yaml"
             config_string = (
                 f"-j {self.config.n_processes} -i {self.config.input_collections_defects} "
-                f"-i {self.config.calib_collection} "
                 "--register-dataset-types "
                 f"{self.config.config_options_defects}"
             )
             exposure_ids = exposure_ids_dict["DARK"] + exposure_ids_dict["FLAT"]
-        elif image_type == "PTC":
-            pipe_yaml = "cpPtc.yaml"
+        elif image_type == "PTC" or image_type == "GAIN":
+            if image_type == "PTC":
+                pipe_yaml = "cpPtc.yaml"
+            else:
+                pipe_yaml = "cpPtc.yaml#gainFromFlatPairs"
             config_string = (
                 f"-j {self.config.n_processes} -i {self.config.input_collections_ptc} "
-                f"-i {self.config.calib_collection} "
                 "--register-dataset-types "
                 f"{self.config.config_options_ptc}"
             )
             exposure_ids = exposure_ids_dict["FLAT"]
         else:
             raise RuntimeError(
                 "Invalid image or calib type {image_type} in 'call_pipetask' function. "
-                "Valid options: ['BIAS', 'DARK', 'FLAT', 'DEFECTS', 'PTC']"
+                "Valid options: ['BIAS', 'DARK', 'FLAT', 'DEFECTS', 'PTC', 'GAIN']"
```

Comment:
Se my comment above about having a class attribute with this information in it. You should not have to be updating text etc every time you add a new image type.